.editorconfig

```diff
@@ -0,0 +1,9 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
```

Comment:
nit, I would start with the default editorconfig. You can generate one by running `dotnet new editorconfig` from the root of the repo.